rllib/agents/ppo/tests/test_ppo.py

```diff
@@ -149,9 +158,16 @@ def test_ppo_compilation_and_schedule_mixins(self):
 
     def test_ppo_exploration_setup(self):
         """Tests, whether PPO runs with different exploration setups."""
-        config = copy.deepcopy(ppo.DEFAULT_CONFIG)
-        config["num_workers"] = 0  # Run locally.
-        config["env_config"] = {"is_slippery": False, "map_name": "4x4"}
+        config = (
+            ppo.PPOConfig()
+            .environment(
+                env_config={"is_slippery": False, "map_name": "4x4"},
```

Comment:
should we also specify the environment name/class here?

Reply:
Yes, this is done via the `env` key and already works in this PR. But keep in mind that you don't have to set all args in any call to any of the `TrainerConfig`'s methods. Only those args that are non-None will be overridden.
You can also set `env` in the `build()` call, but we may disallow that in the future to have really just one way of doing things (reduce confusion in our APIs).